webui/src/pages/auth/groups/group/members.jsx

```diff
@@ -26,15 +26,41 @@ const GroupMemberList = ({ groupId, after, onPaginate }) => {
     const [refresh, setRefresh] = useState(false);
     const [showAddModal, setShowAddModal] = useState(false);
     const [attachError, setAttachError] = useState(null);
-
+    const [allUsers, setAllUsers] = useState([]);
     const {results, loading, error, nextPage} = useAPIWithPagination(() => {
         return auth.listGroupMembers(groupId, after);
     }, [groupId, after, refresh]);
-
     useEffect(() => {
         setAttachError(null);
     }, [refresh]);
 
+    const setAllUsersFromLakeFS = async () => {
+        if (allUsers.length > 0) {
+            return allUsers
+        }
```

Comment:
@itaigilo, isn't there some better way to force only one load, with `useEffect` or something?

Reply:
Probably - 
But I think that if some caching will be added here, it'll change the flow. Not sure though.